httpclient5-cache/src/main/java/org/apache/hc/client5/http/impl/cache/ResponseCachingPolicy.java

```diff
@@ -231,8 +231,18 @@ public boolean isResponseCacheable(final ResponseCacheControl cacheControl, fina
             return false;
         }
 
-        // calculate freshness lifetime
         final Duration freshnessLifetime = calculateFreshnessLifetime(cacheControl, response);
+
```

Comment:
@arturobernalg I cannot find `immutable` cache control directive mentioned anywhere in the RFC 9111. Am I missing something?